ospo-mindmap/Content/ospomindmap.md

```diff
@@ -70,21 +84,80 @@ development team of the Linux Project
+  - Publishing a new open source project
+  - Contributing in personal time
+  - Signing contribution agreements (e.g. CLA or DCOs)
+- Policy on making open source business decisions
+  - Risk assessment and acceptance  
+  - Business alignment
+
+#### Processes
+
+- Creating Open Source
+  - Contributing to third-party OSS projects during work time
+    - Which copyright statement to use for contributions on behalf of the organization
+    - What to do if there is already a copyright notice in the code
+    - What to do if there is no copyright notice in the code
+    - What to do if the community object to any changes to the copyright notices
+  - Contributing to third-party OSS projects during personal time
+  - Publishing a new open source project
+    - Check if the new project name is already in use in the community or is trademarked
+  - Sponsoring Open Source
+    - Sponsoring third-party projects
+    - Sponsoring organization projects or individual members
+    - Sponsoring events
+  - Signing contribution related agreements
+    - Contributor license agreement
+    - Developer certificate of origin
+  - Shutting down a new open source project
+- Using Open Source
+  - Using an open source project as a standalone tool
+  - Including open source snippets or files
+  - Including open source packages
+  - Forking an open source project
+  - Inventorying open source usage incl. packages, licensing and communities
+- Deciding on Open Source
+  - Drive Open Source Review Board with senior leaders from relevant stakeholder to make decisions on open source
+  - Drive Open Source Working Group with representatives / interested stakeholders throughout the organization with the aim of improving open source at the organization
+- Marketing and/or communicating Open Source
+  - Internal messaging
+    - Maintaining internal wiki outlining policy, processes, tooling and best practices
+  - External messaging
+    - Organization's open-source principles
+    - Simplified/redacted version of open source wiki
+    - Organization GitHub ReadMe
+    - Open Source landing page / portal
+    - Obligation fulfillment
+    - Organizing community events
+- Educating people
+  - Open source awareness which includes details on open source policy, processes, tools and the basics of open source licensing, community and contributions
+  - Guidelines and best practices for using of, contributing to, complying with open source
+  - Training on open source project management
+  - New employee orientation
+  - Checklist for product teams
+  - Checklist for developers
+  - Checklist for SW procurement
+  - Executing a mentorship for open source leaders and/or organization leaders
+  - Organizing events such as brown bag seminars (with invited speakers)
+  - Assist speakers for coding forums / conferences
+
+### 📈 Prioritize and Drive Open Source Upstream Development
+
+#### Create Educational Programs
 
 - Good OSS hygiene
 - Technical programs
```

Comment:
Add node: "How to contribute other than code" and under that, e.g. 
- How to contribute code review
- How to contribute documentation
- How to contribute testing
- Etc.

Reply:
Added the following:

  - How to do non-code contributions
    - How to contribute code review
    - How to contribute documentation
    - How to contribute testing
    - How to contribute to project management